src/Desktop.Analyzers/UnitTests/DoNotUseInsecureDTDProcessingUseXmlReaderForXPathDocumentTests.cs

```diff
@@ -0,0 +1,366 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.UnitTests;
+using Microsoft.CodeAnalysis.CSharp;
+using Xunit;
+
+namespace Desktop.Analyzers.UnitTests
+{
+    public partial class DoNotUseInsecureDTDProcessingAnalyzerTests : DiagnosticAnalyzerTestBase
+    {
+        private DiagnosticResult GetCA3075XPathDocumentCSharpResultAt(int line, int column, string name)
+        {
+            return GetCSharpResultAt(line, column, CA3075RuleId, string.Format(CA3075LoadXmlMessage, name, ".ctor"));
+        }
+
+        private DiagnosticResult GetCA3075XPathDocumentBasicResultAt(int line, int column, string name)
+        {
+            return GetBasicResultAt(line, column, CA3075RuleId, string.Format(CA3075LoadXmlMessage, name, ".ctor"));
+        }
+
+        [Fact]
+        public void UseXPathDocumentShouldGenerateDiagnostic()
+        {
+            VerifyCSharp(@"
+using System.Xml;
+using System.Xml.XPath;
+
+namespace FxCopUnsafeXml
```

Comment:
Even in sample code, don't mention FxCop, which we are working to replace. Perhaps in this case, just
```
namespace UnsafeXPath
```